alvr/client_openxr/src/lib.rs

```diff
@@ -139,6 +135,7 @@ pub fn entry_point() {
     xr_entry.initialize_android_loader().unwrap();
 
     let available_extensions = xr_entry.enumerate_extensions().unwrap();
+    alvr_common::info!("OpenXR available extensions: {available_extensions:#?}");
```

Comment:
Especially this one I would expect to be quite spammy so it should probably be a debug log instead

Reply:
This is logged before connection, so it's just printed to logcat

Reply:
The Quest runtime already prints the list before, but our log is a little cleaner